roles/system_setup/tasks/install-node-deps-rpm.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Node dependencies - Set DNF proxy
+  template:
+    src: dnf_proxy.j2
+    dest: /etc/dnf/dnf.conf
+    mode: 0644
+  become: true
+  when: rpm_proxy is defined and rpm_proxy | length > 0
+
+- name: Node dependencies - RPM setup
+  tags:
+    - node_deps
+  become: true
+  block:
+    - name: node dependencies - delete unused repos
+      ansible.builtin.file:
+        name: "{{item}}"
+        state: absent
+      with_items:
+        - /etc/yum.repos.d/fedora-updates-modular.repo
+        - /etc/yum.repos.d/fedora-cisco-openh264.repo
```

Comment:
the more I think about this, the more I wonder if we should be nuking repos on behalf of the OS admin. I'm not sure the origin of these.

Reply:
this is not a blocker, we can handle removal in a separate PR

Reply:
My thought is we're doing it and we probably have a reason why we're doing it 🤷‍♀️ so until someone complains I'm leaving it in.